### DISCUSSION TITLE
EulerA instead of DDIM for scheduler?

### DISCUSSION THREAD

**Discussion by chakalakasp** [d8ahazard/sd_dreambooth_extension#407] [General]
Was curious why it was switched -- does EulerA give better performance as a scheduler?

I'm struggling to get good results from the same datasets that used to give me great results from a when I was using your old version from a month ago, before the big UI update.  I'm trying to narrow down what the variable might be.

**Comment by wktra** [comment]
> Was curious why it was switched -- does EulerA give better performance as a scheduler?
> 
> I'm struggling to get good results from the same datasets that used to give me great results from a when I was using your old version from a month ago, before the big UI update. I'm trying to narrow down what the variable might be.

it's been mentioned in previous issues that the scheduler when creating a model does nothing yet.
